TCAT/Utilities/SearchTableViewHelpers.swift

```diff
@@ -244,6 +246,19 @@ class SearchTableViewManager {
         }
         return sectionIndexDictionary
     }
+    
+    func updateShortcutItems(favorites: [Any]) {
+        var shortcutItems = [UIApplicationShortcutItem]()
+        if let favorites = favorites as? [Place] {
+            for item in favorites {
+                let data = NSKeyedArchiver.archivedData(withRootObject: item)
```

Comment:
here also seems to have same code as code you put in `AppDelegate` > `didFinishLaunchingWithOptions`. so when create a function for this code i would use the function instead of having repeated bodies of code

Reply:
Yeah, let's have this written once. Maybe it can live in the AppDelegate. You can call it doing something like

`
guard let appDelegate = UIApplication.shared.delegate as? AppDelegate
appDelegate.updateShortcutItems(...)
`

Reply:
@mattbarker016 I disagree. It should probably be a static method somewhere else, but let's try and keep AppDelegate clean. Maybe in a utility file or a new `Shortcuts.swift` file

Reply:
Added the functions to `AppShortcuts.swift` in Utilities